config/initializers/sufia.rb

```diff
@@ -8,13 +8,15 @@
 
   config.max_notifications_for_dashboard = 5
 
+  # Make sure that the default choice is first in the list.  We tried several
+  # ways to specify a default without success. --RC/SC
   config.cc_licenses = {
+    'Attribution-NonCommercial-ShareAlike 3.0 United States' => 'http://creativecommons.org/licenses/by-nc-sa/3.0/us/',
+    'Attribution-NonCommercial 3.0 United States' => 'http://creativecommons.org/licenses/by-nc/3.0/us/',
+    'Attribution-NonCommercial-NoDerivs 3.0 United States' => 'http://creativecommons.org/licenses/by-nc-nd/3.0/us/',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.